oop/src/views/MoveInfoPage.vue

```diff
@@ -480,8 +480,21 @@ export default {
         validations.addressLines = {
           $each: {
             value: {
-              specialCharacterValidator,
-              required
+              required,
+              specialCaseValidator: (value, addressLine) => {
```

Comment:
Can you define this validator function so that it isn't inline? Example: https://github.com/bcgov/common-lib-vue/blob/main/src/components/DateInput.vue#L88-L93

Also can you rename the validator to something more descriptive? Something like `addressLineWithCommaValidator`.